acceptable/tests/test_main.py

```diff
@@ -372,166 +375,171 @@ def test_render_markdown_success(self):
         args = main.parse_args(
             ['render', 'examples/api.json', '--name=SERVICE'])
 
```

Comment:
The diffs below this point look noisy, but merely consist of adding this after each call to main.parse_args():

    with contextlib.closing(args.metadata):

and indenting everything in the method after that point.